#!/usr/bin/env python
PACKAGE = "obstacle_detection_2019"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ransacNum", int_t, 0, "An Integer parameter", 500, 1, 2000)
gen.add("ransacDistanceThreshold", double_t, 0, "A double parameter", .1, .01, 1)
gen.add("ransacEpsAngle", double_t, 0, "A double parameter", 15, .01, 90)
gen.add("estimateCandidateY", double_t, 0, "A double parameter", .3, -0.5, 1)
gen.add("estimateCameraHeight", double_t, 0, "A double parameter", .4125, 0, 1)
gen.add("estimateGroundThreshold", double_t, 0, "A double parameter", .2, -0.5, 1)
gen.add("estimateHeightThreshold", double_t, 0, "A double parameter", 1.0, 0, 5)

exit(gen.generate(PACKAGE, "obstacle_detection_2019", "convCamData"))